app/Services/MentionsService.php

```diff
@@ -122,6 +122,7 @@ public function mapAttribute(Attribute $attribute)
         }
 
         $this->text = (string) $attribute->value;
+        $this->text = Attributes::parse($attribute);
```

Comment:
You're setting $this->text twice, and calling `Attributes::parse()` twice, which seems very redundant (and bad for performance). Can you clean up the code to only call these things once? And also add a comment in the code as to why we are calling the methods in this order, and not the old order :)